specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-07-01/managedClusters.json

```diff
@@ -6145,6 +6149,20 @@
         }
       }
     },
+    "AgentPoolSecurityProfile": {
```

Comment:
Seems `AgentPoolSSHAccess` is not included in this profile, expected?

Reply:
Yes, this feature is still in preview